### ISSUE TITLE
OCP 4.8 Upgrade Openshift in KLAB

### ISSUE THREAD

**Issue by wmhutchison** [BCDevOps/developer-experience#2213]
**Describe the issue**
Run the OCP 4.8 upgrade in KLAB. Document if/where it gets stuck and any steps to work around it. Try and determine how disruptive the changes are to the API, Router, and hosted apps.
(Blocked by: Hard Prune in KLAB)

**Additional context**
For all of the involved playbooks, checkout and use the ocp48-upgrade branch to run them from. These branches will be merged into main if no further changes are required.
CCM changes were merged, KLAB needs only to have its PR merged for OCP 4.8 changes to be applied there.

**Definition of done**
- [x] Upgrade `oc` CLI on util server
- [x] KLAB upgraded to OCP 4.8
- [x] MachineConfig changes applied
- [x] CCM applied
- [x] EnCase updated
- [x] ETCD Backup updated
- [x] Nagios updated
- [x] Pruner updated
- [x] vROps updated
- [x] Logging updated (5.3 channel)
- [x] Sample Operator updated

**Comment by tbaker1313** [comment]
Completed successfully.
